### DISCUSSION TITLE
[drive] How to get name after moveToDisk ?

### DISCUSSION THREAD

**Discussion by Barbapapazes** [adonisjs/core#3127] [Help]
Hello, 

If we look at [this function](https://github.com/adonisjs/bodyparser/blob/develop/src/Multipart/File.ts#L209), we can see that a new filename is given but is there a way to get it? Or we must pass the filename through options to customize it in the same way?

Thanks!

**Comment by thetutlage** [comment]
You can access it from the file instance https://docs.adonisjs.com/guides/file-uploads#filename
